app/src/main/res/layout/fragment_breathless.xml

```diff
@@ -82,12 +82,12 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@id/cause_recycler_view" />
 
-        <ProgressBar
+        <org.coepi.android.ui.extensions.ProgressBarViewExtension
             android:id="@+id/progress"
             style="@style/screen_progress_indicator"
             isVisible="@{vm.isInProgress}"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_width="match_parent"
```

Comment:
`match_parent` shouldn't be used in `ConstraintLayout`. It should be `0dp` (you're already using constraints to pin the edges to the parent).